docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+|  library_name               | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Name of a sequencing library made from a molecular sample or a sample pool (multiplex sequencing).                                                                                                                                                                                                                                                                          |                      |  
+|  fileName       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Name of the file.                                                                            |                      |  
+|  fileSize       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Size of the file, in bytes.                                                                  |                      |  
+|  fileMd5sum     | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Compute the md5sum of the file. This must match what is computed when the file is uploaded.  |  |  |
+|  fileType       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Data format of sequencing files.                                                             |  BAM, FASTQ          |  
+|  fileAccess     | files |  ![Required](/assets/submission/dictionary-required.svg)  |  The permission level of a file.                                                              |  open, controlled    |
+|  dataType       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Descriptor of the type of file being submitted.                                              |  Submitted Reads     | 
+
+
+## Validating Metadata Payloads
+
+It is very important that molecular data is submitted with valid metadata. As a helpful tool during metadata preparation, we have prepared a validation client that can be run on your data locally before submitting data officially. 
+
+Validation will help you ensure your data is formatted correctly (with accurate identifier assignments between metadata and molecular data files) and that your submission goes smoothly while also helping the DCC ensure that the downstream [Analysis Pipelines](/docs/analysis-workflows/analysis-overview) will function seamlessly.  
+
+This validation tool will check: 
+- Data submitted in the metadata payload and BAM/FASTQ files aligns.
+- Each submission is for a single sample.
+- Each payload is formatted according to the [data validation rules](/docs/submission/molecular-data-prep#file-and-data-validation-rules) stated above.
+
+
+### Installing the Validation Client 
+
+<!---  Tabs start here -->
```

Comment:
wonder if we can get Ciaran to put a box around the tab content here...
Because there are steps for these ones (not just code blocks like above) it's hard to understand what content it with what tab.

Reply:
yea we can

Reply:
@kcullion can you make a separate ticket for that?

Reply:
nvm done!!